.rubocop.yml

```diff
@@ -12,9 +12,6 @@ RSpec/MultipleExpectations:
 RSpec/NestedGroups:
   Max: 5
 
-RSpec/NotToNot:
```

Comment:
Why was this removed?

Reply:
That was accidental, I was actually just fixing that.